sql-join/magic-mallrats.sql

```diff
@@ -0,0 +1,11 @@
+select "films"."title",
+"customers"."firstName",
+"customers"."lastName",
+"rentals"."customerId",
+"rentals"."rentalId",
+"inventory"."inventoryId"
```

Comment:
double check the instructions for each SQL file. Only specific columns were requested.